client/benches/tps/utils.rs

```diff
@@ -171,9 +172,7 @@ impl MeasurerUnit {
     fn spawn_event_counter(&self) -> thread::JoinHandle<Result<()>> {
         let listener = self.client.clone();
         let (init_sender, init_receiver) = mpsc::channel();
-        let event_filter = PipelineEventFilter::new()
-            .for_entity(PipelineEntityKind::Block)
-            .for_status(PipelineStatusKind::Committed);
+        let event_filter = BlockEventFilter::default().for_status(BlockStatus::Applied);
```

Comment:
there is both `Committed` and `Applied` events. Please check again it with this in mind

Reply:
i see now, not sure that we need two separate events for this because this look like implementation detail
i would prefer single `Committed` event after changes are reflected in state

Reply:
`Block::Applied` signifies that future queries are guaranteed to return this block.

rationale why I did it like this:
1. block is committed once it transitions to `CommittedBlock`
2. we also broadcast to other peers that block was committed before it actually gets applied to wsv
3. however, querying for this block might fail before it's actually applied to wsv
4. There might be delays between committing and it reflected in peer's wsv

I'm willing to rename `BlockStatus::Applied` to `Block::Committed` if you think it's useless

Reply:
Ok, in this case having two distinct states is fine to me. 
Maybe in this case we can use the following naming `Confirmed` and `Committed`?

Reply:
@0x009922 @DCNick3 your thoughts here?

Reply:
I think `Applied` (as in applied to WSV) is fine. The distinction between Committed and Applied feels a bit weird, but if it is a significant distinction to expose to users, I think the naming is fine.